packages/client/components/DashNavList/DashNavMenu.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {useHistory} from 'react-router'
+import {PALETTE} from '../../styles/paletteV3'
+import {Menu} from '../../ui/Menu/Menu'
+import {MenuContent} from '../../ui/Menu/MenuContent'
+import {MenuItem} from '../../ui/Menu/MenuItem'
+import LeftDashNavItem from '../Dashboard/LeftDashNavItem'
+
+const StyledLeftDashNavItem = styled(LeftDashNavItem)<{isViewerOnTeam: boolean}>(
+  ({isViewerOnTeam}) => ({
+    color: isViewerOnTeam ? PALETTE.SLATE_700 : PALETTE.SLATE_600,
+    borderRadius: 44,
+    paddingLeft: 15
+  })
+)
+
+type Props = {
+  orgId: string
+}
+
+const DashNavMenu = (props: Props) => {
+  const {orgId} = props
+  const history = useHistory()
+  const menuItems = [
+    {
+      label: (
+        <>
+          Plans & Billing •&nbsp;<span className='text-sky-500'>Upgrade</span>
```

Comment:
I think the Upgrade label should be conditional. I’d either remove it altogether or make the `• Upgrade` bit conditional. It shows if the org is on the Starter tier.

Reply:
Thanks, updated!